### DISCUSSION TITLE
ES外表找不到索引

### DISCUSSION THREAD

**Discussion by panxianhaoo** [apache/doris#14326] [A - General / Q&A]
ES版本：7.17.3（使用官方Helm部署在K8s上）
Doris版本：1.1.3

ES建索引语句
![image](https://user-images.githubusercontent.com/30815101/202134472-90bd4e8c-3e27-4f30-9dfa-04b26344f506.png)

Doris建外表语句
CREATE EXTERNAL TABLE `pxh_es_table` (
  `userid` bigint(20),
  `username` varchar(100),
  `usercontent` varchar(20)
) ENGINE=ELASTICSEARCH
PROPERTIES (
"hosts" = "http://172.16.11.121:32124/",
"index" = "pxh_es_test",
"user" = "",
"password" = ""
);

索引是存在的
![image](https://user-images.githubusercontent.com/30815101/202135595-e9030ff8-0578-4799-a90d-c54858c6d0d7.png)

但是报错说找不到索引
![image](https://user-images.githubusercontent.com/30815101/202135734-3ac5c648-8046-4eaf-b52d-d829bffda7eb.png)

**Comment by stalary** [comment]
Please provide http://172.16.11.121:32124/pxh_es_test/_mapping response data.
